client/landing/jetpack-cloud/components/threat-item/index.tsx

```diff
@@ -93,9 +95,10 @@ const ThreatItem: React.FC< Props > = ( {
 		[ threat ]
 	);
 
+	const currentRoute = useSelector( getCurrentRoute );
 	const onOpenTrackEvent = useTrackCallback( noop, 'calypso_jetpack_scan_threat_itemtoggle', {
 		threat_signature: threat.signature,
-		section: window.location.pathname.includes( '/scan/history' ) ? 'History' : 'Scanner',
+		section: currentRoute && currentRoute.includes( '/scan/history' ) ? 'History' : 'Scanner',
```

Comment:
Should we define a section at all if `currentRoute` is not defined?

Reply:
> Probably I missed something, but I can't see the track events, no t.gif in my Network

To see an event, you need at least one threat in the History section. So you need to fix or ignore one to make it appear in that section. Then, clicking the threat will open it and track the event.

![image](https://user-images.githubusercontent.com/3418513/83643128-0a959f80-a586-11ea-82f8-65520715d009.png)



> Another thing, is this okay? The history seems empty.
>
> ![image](https://user-images.githubusercontent.com/9832440/83630047-1045b200-a593-11ea-8628-a9789c3f8d8c.png)

Yes. This is not great but it's have we have now when a user hasn't fixed/ignored any threat in the past. So, it's okay but we should improve it.

> And this is the API response:
> ![image](https://user-images.githubusercontent.com/9832440/83630090-23f11880-a593-11ea-8fde-559329d3a0db.png)
>
> is it okay?

Not sure about this one. It looks like the response we got when we had a fake endpoint. I'll investigate this one. Thanks @cleacos !

Reply:
Good point. I'll remove the property if `currentRoute` is not defined.

Reply:
I updated this @monsieur-z . Whenever you have time, I would love to have your eyes on this again :)